terasoluna-gfw-functionaltest-env/pom.xml

```diff
@@ -1,55 +1,72 @@
-                        <phase>install</phase>
-                        <goals>
-                            <goal>install-file</goal>
-                        </goals>
-                        <configuration>
-                            <file>${basedir}/3rdparty/com/oracle/ojdbc7/${ojdbc7.version}/ojdbc7-${ojdbc7.version}.jar</file>
-                            <pomFile>${basedir}/3rdparty/com/oracle/ojdbc7/${ojdbc7.version}/ojdbc7-${ojdbc7.version}.pom</pomFile>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-    <properties>
-        <project.root.basedir>${project.parent.basedir}</project.root.basedir>
-    </properties>
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>terasoluna-gfw-functionaltest-env</artifactId>
+  <packaging>jar</packaging>
+  <parent>
+    <groupId>org.terasoluna.gfw.functionaltest</groupId>
+    <artifactId>terasoluna-gfw-functionaltest</artifactId>
+    <version>5.4.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <name>TERASOLUNA Server Framework for Java (5.x) - Functionality Test Application - Environment Layer</name>
+  <description>Environment dependent layer of Functionality Test Application using TERASOLUNA Server Framework for Java (5.x)</description>
+  <build>
+    <finalName>${project.env.buildFinalName}</finalName>
+    <resources>
+      <resource>
+        <directory>${project.env.main.directory}/resources</directory>
+        <excludes>
+          <exclude>**/*.env.ftl</exclude>
+        </excludes>
+      </resource>
+    </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <configuration>
+          <encoding>${encoding}</encoding>
+          <overwrite>true</overwrite>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-install-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>install-ojdbc7</id>
+            <phase>install</phase>
+            <goals>
+              <goal>install-file</goal>
+            </goals>
+            <configuration>
+              <file>${basedir}/3rdparty/com/oracle/ojdbc7/${ojdbc7.version}/ojdbc7-${ojdbc7.version}.jar</file>
+              <pomFile>${basedir}/3rdparty/com/oracle/ojdbc7/${ojdbc7.version}/ojdbc7-${ojdbc7.version}.pom</pomFile>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>xml-maven-plugin</artifactId>
+        <version>${xml-maven-plugin.version}</version>
+        <configuration>
+          <indentSize>2</indentSize>

```

Comment:
- [ ] Is `indentSize` really necessary? It seems to be inherited from parent pom.